src/main/java/org/opensearch/security/filter/SecurityRestFilter.java

```diff
@@ -26,7 +26,10 @@
 
 package org.opensearch.security.filter;
 
+// CS-SUPPRESS-SINGLE: RegexpSingleline Extension is used to refer to certificate extensions, keeping this rule disable for the whole file
```

Comment:
Please activate/deactivate this suppression only in the places where you are making changes.  There is only one other cluster of references to extensions in this file.